include/ck_tile/ops/epilogue/chainer/cshuffle_epilogue_schedule.hpp

```diff
@@ -0,0 +1,122 @@
+        }
+        else if constexpr(std::is_same_v<ScheduleTag, RowColQuantScheduleTag>)
+        {
+            // RowCol quantization schedule with tensor windows
+            // Schedule: Slice -> ScaleWindow -> Cast -> PrepC -> ApplyD -> Store -> MoveWindows
+            static_assert(sizeof...(args) == 2,
+                          "RowColQuantSchedule requires exactly 2 scale tensor arguments");
+            return make_graph<NumAccess>(
+                make_node<SliceEpilogue<typename BaseOp::SFC,
+                                        typename BaseOp::CWarpDstr,
+                                        BaseOp::NumMXdlPerWavePerShuffle,
+                                        BaseOp::NumNXdlPerWavePerShuffle,
+                                        BaseOp::MPerIterationShuffle,
+                                        BaseOp::NPerIterationShuffle>>(),
+                make_node<ScaleWindowEpilogue<typename BaseOp::SFC>>(std::forward<Args>(args)...),
+                make_node<CastLdsEpilogue<typename BaseOp::ODataType>>(),
+                make_node<PrepCTensorEpilogue<typename BaseOp::TileEncodingPattern>>(),
+                make_node<ApplyDEpilogue<typename Problem::CDElementwise, Problem::NumDTensor>>(),
+                make_node<StoreToDramEpilogue<Problem::MemoryOperation>>(),
+                make_node<MoveWindowsEpilogue<typename BaseOp::SFC, Problem::NumDTensor>>());
+        }
+        else if constexpr(std::is_same_v<ScheduleTag, TensorQuantScheduleTag>)
+        {
+            // Tensor quantization schedule with scalar values
+            // Schedule: Slice -> ScaleScalar -> Cast -> PrepC -> ApplyD -> Store -> MoveWindows
+            static_assert(sizeof...(args) == 2,
+                          "TensorQuantSchedule requires exactly 2 scalar arguments");
+            return make_graph<NumAccess>(
+                make_node<SliceEpilogue<typename BaseOp::SFC,
+                                        typename BaseOp::CWarpDstr,
+                                        BaseOp::NumMXdlPerWavePerShuffle,
+                                        BaseOp::NumNXdlPerWavePerShuffle,
+                                        BaseOp::MPerIterationShuffle,
+                                        BaseOp::NPerIterationShuffle>>(),
+                make_node<ScaleScalarEpilogue<typename BaseOp::SFC>>(std::forward<Args>(args)...),
+                make_node<CastLdsEpilogue<typename BaseOp::ODataType>>(),
+                make_node<PrepCTensorEpilogue<typename BaseOp::TileEncodingPattern>>(),
+                make_node<ApplyDEpilogue<typename Problem::CDElementwise, Problem::NumDTensor>>(),
+                make_node<StoreToDramEpilogue<Problem::MemoryOperation>>(),
+                make_node<MoveWindowsEpilogue<typename BaseOp::SFC, Problem::NumDTensor>>());
+        }
+        else
+        {
+            static_assert(sizeof(ScheduleTag) == 0, "Unknown schedule tag");
```

Comment:
Why checking size ? IMHO this wouldn't work. Wouldn't this work: 
```suggestion
            static_assert(false, "Unknown schedule tag");
```